src/components/SeatNumbers.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import {SeatBook} from '../Search.styled'
+const SeatNumbers = () => {
+  return (
+      <SeatBook>
```

Comment:
Make use of array and loop through it. Like arr = [1,2,3, ... 10] and `arr.map(item => (<li>{item}</li>))`